### PR TITLE
Blur & Fantasy trades

### PR DESCRIPTION
**Pull request by hildobby**
### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [ ] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For new spell(s)
If you are building new spell(s), please provide the following information:

- **Spell name(s):** *schema.table_name*
- **Description:** [Detailed description of the new spell(s) and their purpose]
- **Who are the new spell(s) for?** [Internal team or general community]
- **How will the new spell(s) be used downstream?** [Description of downstream usage]
- **Implementation details:** [Information on how the spell(s) are implemented]
- **Test instructions:** [How to test the new spell(s)]
- **Related issue(s):** [Link to related issues, if any]

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** [Description of the changes made]

---

### For bug fixes
If you are fixing a bug, please provide the following information:

- **Description:** [Description of the bug fix]
- **Steps to reproduce:** [How to reproduce the bug]
- **Implementation details:** [Information on how the bug was fixed]
- **Test instructions:** [How to test the fix]
- **Related issue(s):** [Link to related issues, if any]

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

File: models/_sector/nft/trades/platform_views/fantasy_trades_view.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+{{ config(
+        schema = 'fantasy',
+        alias = 'trades',
+        
+        materialized = 'view',
+        post_hook='{{ expose_spells(\'["blast"]\',
+                                    "project",
+                                    "nftrade",
```
Comment (0xRobin) [inline models/_sector/nft/trades/platform_views/fantasy_trades_view.sql:9]:
```suggestion
                                    "fantasy",
```

**Review by 0xRobin — APPROVED** [review]
approving, but will hold of on merging as we're in the middle of moving the NFT lineage to it's own subproject. (https://github.com/duneanalytics/spellbook/pull/6090)

**Comment by hildobby** [comment]
> approving, but will hold of on merging as we're in the middle of moving the NFT lineage to it's own subproject. (#6090)

ah alright, then i'll also add blur on blast in here in the meantime. any chance of it getting merged by the time blast is announced to be on dune? because that's what i told the teams 😬

**Comment by jeff-dude** [comment]
@aalan3 recommended proceeding here, then rebase other PR which moves the spells

**Pull request merged**